src/content/docs/usage/workspaces.mdx

```diff
@@ -64,66 +105,90 @@ daytona create --code
    daytona create https://github.com/daytonaio/daytona.git
    ```
 
-2. Wait for Daytona to create the Workspace.
+   Upon running this command, Daytona will clone the specified Git repository and set up a new Workspace based on its contents. You will see progress information in the terminal, indicating the process of the Workspace creation. Once complete, the Workspace will be ready for use with the specified repository.
+
+   ```text
+      WORKSPACE | ✓ Request submitted
+      WORKSPACE | ✓ Creating workspace daytona
+      daytona   | Creating project daytona
+      daytona   | Pulling image...
+      daytona   | Pulling from daytonaio/workspace-project
+      ...
+      daytona   | Project daytona created
+      daytona   | Starting project daytona
+      daytona   | Project daytona started
+   ```
 
 <Aside type="tip">
 You can set the Project Builder at Workspace creation time using command-line flags.
 These flags only apply when creating Workspaces with a single Project.
--   To use the Custom Image Builder, set both `--custom-image` and `--custom-image-user` flags.
--   To use the Dev Container Builder, set the`--devcontainer-path` flag to the Dev Container configuration path within the repository.
--   To explicitly specify the Builder, set the`--builder` flag.
+- To use the Custom Image Builder, set both `--custom-image` and `--custom-image-user` flags.
```

Comment:
@stefanicjuraj does it make sense to remove most of this, and jsut say for more on builders go to builders (link) and show the info there?